### PR TITLE
Minor polishing and upgrades

### PR DESCRIPTION
**Pull request by erikkemperman**
Some minor polishing. Note, mutual dependency on a generator [PR](https://github.com/jhipster/generator-jhipster/pull/7428).

Both PRs have a temporary commit at the tip of the branch which should _not_ be merged but are there to make Travis pass.

https://travis-ci.org/erikkemperman/jhipster/builds/363105228

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,8 +34,8 @@ env:
     - SPRING_JPA_SHOW_SQL=false
 
     # if JHIPSTER_BRANCH value is release, use the release from NPM
-    - JHIPSTER_REPO=https://github.com/jhipster/generator-jhipster.git
-    - JHIPSTER_BRANCH=master
+    - JHIPSTER_REPO=https://github.com/erikkemperman/generator-jhipster.git
```
Comment (jdubois) [inline .travis.yml]: Sorry but this is a mistake I think

**Comment by jdubois** [comment]
Merging this manually...

**Comment by jdubois** [comment]
Merged manually

File: .travis.yml (hunk 1 above)
Comment (erikkemperman) [reply .travis.yml]: Yes and no -- see the discussion on the accompanying PR on the generator repo:
https://github.com/jhipster/generator-jhipster/pull/7428#discussion_r179921288

I know this is annoying to merge this way, but I don't know how else to make Travis pass on mutually dependent PRs across two repos!

**Comment by erikkemperman** [comment]
Thanks!

**Pull request closed without merging**